contracts/controller/src/lib.rs

```diff
@@ -74,6 +78,17 @@ pub struct PriceJsonList {
     pub price_list: Vec<Price>,
 }
 
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+pub struct ActionStatus {
+    pub withdraw: bool,
```

Comment:
@dmytro-shash seems that none of this fields should be public. Also please order it according to user flow
supply
withdraw
borrow
repay
liquidate